tests/test_rtdc_feat_anc_plugin.py

```diff
@@ -55,6 +55,12 @@ def compute_non_scalar_plugin_feature(rtdc_ds):
     return {"image_gauss_filter": image_gauss_filter}
 
 
+def compute_mask_plugin_feature(rtdc_ds):
+    """Basic plugin method creates mask data (boolean data)"""
+    mask_feat = np.logical_not(rtdc_ds["mask"])
```

Comment:
Maybe use a different name than `mask_feat`. It is too broad.

Reply:
Can you recommend a name?

Reply:
Sorry for the overlook!